### PR TITLE
EslintConfiguration

### PR DESCRIPTION
**Pull request by anastasiiaderkach1996**

### CODE REVIEW

File: client/src/components/ItemList.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
 import Point from './Point';
```
[the remark this answers is not in the document]
Comment (anastasiiaderkach1996) [reply client/src/components/ItemList.js]: Fixed

[the remark this answers is not in the document]
Comment (yuriiholskiy) [reply client/src/components/ItemList.js]: Good job!

File: client/.eslintrc.json [hunk 2]
```diff
@@ -20,6 +21,7 @@
   },
   "plugins": ["react"],
   "rules": {
```
Comment (OrestSat) [inline client/.eslintrc.json]: Use these rules 
`"rules": {
        "no-multiple-empty-lines": [
          "error",
          {
            "max": 1
          }
        ],
        "indent": [
            "warn",
            2,
            {"SwitchCase": 1}
        ],
        "linebreak-style": [
            "error",
            "unix"
        ],
        "quotes": [
            "error",
            "single"
        ],
        "semi": [
            "error",
            "always"
        ],
        "react/jsx-uses-vars": "error",
        "react/jsx-uses-react": "error",
        "react-hooks/rules-of-hooks": "error"
    }`

Comment (anastasiiaderkach1996) [reply client/.eslintrc.json]: Fixed 
Then i will changed warn to error

**Pull request closed without merging**